app/gui/src/project-view/components/MarkdownEditor/DocumentationImage.vue

```diff
@@ -34,16 +34,55 @@ const title = computed(() =>
 
 const alt = props.alt ? props.alt : DEFAULT_ALT_TEXT
 
+const isYouTubeVideo = computed(() => {
+  return props.src.match(/https:\/\/www\.youtube\.com\/embed\/[^\/]+/)
```

Comment:
We might also want to support `youtube-nocookie.com` domain, since people often use that for embeds as well.